hw03_finagnon.py

```diff
@@ -0,0 +1,299 @@
+
+#################################################################################
+# Author:Finagnon Adjagbodjou
+# Username: Fin
+#
+# Assignment: hw03
+# Purpose: Learn the purpose of pulling, pushing, etc.
+# Google Doc Link: https://docs.google.com/document/d/1BjZMQNg6d2PG-PlwE5gFo9XBrmMei0aY6B_CtXdtFLg/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements: 
+#
+#
+#################################################################################
+
+import turtle
+import turtle
+import random
+
+def rightturn(shape):
+    """
+    turns the turtle to the right
+    """
+    shape.right(90)
+    shape.forward(5)
+    shape.right(90)
+    pass
+    # ...
+
+
+def leftturn(shape):
+    """
+    turns the turtle to the left
+    """
+    shape.left(90)
+    shape.forward(5)
+    shape.left(90)
+
+    pass
+    # ...
+
+def createrectangle(shape):
+    """
+    creates the starting rectangle in which the turtle will move
+    """
+
+    shape.penup()
+    shape.pensize(20)
+    shape.setpos(-200, 250)
+    shape.pendown()
+    shape.color('black')
+    shape.forward(500)
+    shape.right(90)
+    shape.forward(505)
+    shape.right(90)
+    shape.forward(500)
+    shape.right(90)
+    shape.forward(500)
+
+    pass
+    # ...
+def forwardmovement(shape):
+    """
+    moved the turtle forward
+    """
+    shape.forward(460)
```

Comment:
This function does a single line of code, which makes it meaningless (i.e., you can just call the `shape.forward()` function directly instead of calling your function, which then calls it).